aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -376,6 +648,39 @@ impl AggregatorData {
         }
     }
 
+    pub fn snapshot(&mut self, id: &AggregatorID) -> u64 {
+        let new_id = self.generate_id();
+        let snapshot_id = AggregatorID::ephemeral(new_id);
+        let aggregator = self.aggregators.get(id).expect("Aggregator doesn't exist");
```

Comment:
Should / can we have fallback here as well (and similar places), i.e. return the CODE_ERROR like in expect_ok (maybe customized for ID lookups), since at the end of the day on the executor side we can fall back to aggregator-less execution eventually, so we would be tolerate an ID related bug where look-ups failed. @igor-aptos - what's your thoughts too